storage/test/test_cloudfront.py

```diff
@@ -98,6 +99,44 @@ def test_direct_download(
             # get back an S3 URL.
             assert "s3.amazonaws.com" in engine.get_direct_download_url(_TEST_PATH, "1.2.3.4")
 
+        engine = CloudFrontedS3Storage(
+            context,
+            "failure",
```

Comment:
instead of calling this `failure` can you call this `defaultdomain` and the other one `overridedomain`